languages/kotlin/maintainers.md

```diff
@@ -8,4 +8,7 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+  - Elena Parovyshnaia (Slack: `@Elena Parovyshnaia`, Github: [eparovyshnaya](https://github.com/eparovyshnaya))
+  - Uzi Landsmann (Slack: `@uzilan`, Github: [uzilan](https://github.com/uzilan))
+  - Mike Gehard (Slack: `@mikegehard`, Github: [mikegehard](https://github.com/mikegehard))
+  - Denys M (Slack: `@dector`, Github: [dector](https://github.com/dector))
```

Comment:
Replace Github with GitHub for consistency in all maintainers.md files

Reply:
Done

Reply:
My bad. I thought I copied it from C# template.